specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/workbookTemplateOperations_API.json

```diff
@@ -0,0 +1,129 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "version": "2018-06-17-preview",
+      "title": "WorkbookTemplateClient",
```

Comment:
This exact API is already defined in 2015-05-01 aiOperations_API.  The provider is the same and the definitions are the same.  It looks like this all goes to the same client as well in the readme... is this necessary?  If it was copied from workbookOperations_API.json it looks like that file isn't referenced in the readme.

Reply:
This is referenced in the readme. At first I had not added this file and a whole bunch of validations failed.

Reply:
Fixed it.